generators/server/files.js

```diff
@@ -763,6 +758,11 @@ const serverFiles = {
             condition: generator => generator.authenticationType !== 'oauth2',
             path: SERVER_TEST_SRC_DIR,
             templates: [
+                {
```

Comment:
I ended up keeping this test around for OAuth since the server is still using it's token after login from OAuth.  I just had to change it so that it wasn't replacing the DatabaseAuthenticator

Reply:
Good point, will try to check tonight

Reply:
@JasonTypesCodes Can you check what happens when you correct the assertions in `JWTFilterTest` like ` assertThat(username).isEqualTo("test-user");` on master?

Reply:
@atomfrede Excellent catch.  Basic Auth was still enabled and the bad assertions were letting it slip through.  I've pushed a fix:: https://github.com/jhipster/generator-jhipster-micronaut/commit/fab2a10f776d7726e0eb5c13a5f72e5a18c30bf0

If the CI checks out I'll PR and merge it in.